easy-coding-standard.neon

```diff
@@ -0,0 +1,10 @@
+# this will be placed in doctrine/coding-stanard, easy-accessible for anyone
```

Comment:
It's famously similar to YAML with the notable exception that it uses tabs instead of spaces for indentation. Class autocomplete is something the IDE offers, not a feature of the config file itself.